### DISCUSSION TITLE
OpenGrok: showing wrong search results

### DISCUSSION THREAD

**Discussion by vinaybs1983** [oracle/opengrok#4652] [Q&A]
When searching for `<string>`, openGrok claims to have found it in latest trunnk commit  in .hpp file

But when opening that file, String  is not present there. It is present in the previous  5 commits before . Opengrok should show only in the atest trunk  

Opengrok version : 1.13.17
Docker instance is being used

**Comment by vladak** [comment]
This means that the indexer for that project has not run since the latest changes were pulled or failed when indexing. There is not enough information to tell what happened.
